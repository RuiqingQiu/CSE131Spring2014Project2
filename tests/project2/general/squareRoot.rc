function : float sqrt(float n){
   float f = 1;
   int i = 0; while (i++ < 10){
      f = (f + n/f)/2;
   }
   return f;
}
function : int main(){
   int i = 0; while (++i <= 100){
      cout << sqrt(i) << " ";
      if (i % 10 == 0) { cout << endl; }
   }
   cout << endl;

   return 0;
}
