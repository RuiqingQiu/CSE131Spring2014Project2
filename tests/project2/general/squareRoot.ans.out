1.00 1.41 1.73 2.00 2.24 2.45 2.65 2.83 3.00 3.16 
3.32 3.46 3.61 3.74 3.87 4.00 4.12 4.24 4.36 4.47 
4.58 4.69 4.80 4.90 5.00 5.10 5.20 5.29 5.39 5.48 
5.57 5.66 5.74 5.83 5.92 6.00 6.08 6.16 6.24 6.32 
6.40 6.48 6.56 6.63 6.71 6.78 6.86 6.93 7.00 7.07 
7.14 7.21 7.28 7.35 7.42 7.48 7.55 7.62 7.68 7.75 
7.81 7.87 7.94 8.00 8.06 8.12 8.19 8.25 8.31 8.37 
8.43 8.49 8.54 8.60 8.66 8.72 8.77 8.83 8.89 8.94 
9.00 9.06 9.11 9.17 9.22 9.27 9.33 9.38 9.43 9.49 
9.54 9.59 9.64 9.70 9.75 9.80 9.85 9.90 9.95 10.00 

